keras/models.py

```diff
@@ -741,28 +741,59 @@ def save_weights(self, filepath, overwrite=True):
 
     def compile(self, optimizer, loss,
                 metrics=None,
+                loss_weights=None,
```

Comment:
Loss weights do not make sense for `Sequential` since the model has a single output.

Reply:
Maybe we should warn/assert that len(loss_weight)==1. Maybe somebody wants to multiply loss by a constant or something. It' mainly to make a sequential Model and Sequential interchangeable as much as practicable.

Reply:
When passing multiple loss_weights to this function, a very clear exception is raised:
```
  File "./examples/cifar10_cnn.py", line 68, in <module>
    loss_weights=[1,2])
  File "build/bdist.linux-x86_64/egg/keras/models.py", line 817, in compile
  File "build/bdist.linux-x86_64/egg/keras/engine/training.py", line 697, in compile
ValueError: When passing a list as loss_weights, it should have one entry per model
outputs. The model has 1 outputs, but you passed loss_weights=[1, 2]
```

Reply:
I would definitely exclude `loss_weights` from the `Sequential` API. There is no use case for it, and it is likely to confuse some people.

Reply:
Excluded. While at it, I (accidentally) removed previously present Sequential.loss_weights attribute as well, but it seems like the right thing to do.